apps/omg_eth/test/support/snapshot_contracts.ex

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule Support.SnapshotContracts do
```

Comment:
here again, unsure - I decided to split it out (places where this was used used `Eth` already anyway, so coupling increase isn't huge) and put under `omg_eth` once.